packages/extension/src/view/devtools/pages/learning/helpCenter/index.tsx

```diff
@@ -17,19 +17,28 @@
 /**
  * External dependencies.
  */
-import { Help } from '@google-psat/design-system';
 import React from 'react';
+import { Help, SIDEBAR_ITEMS_KEYS } from '@google-psat/design-system';
+
+/**
+ * Internal dependencies.
+ */
+import { FEATURED_ITEMS } from '../constants';
+import ExternalLinkPanel from '../../shared/externalLinkPanel';
 
 const HelpCenter = () => {
+  const description =
+    FEATURED_ITEMS.find(
+      (item) => item.sidebarKey === SIDEBAR_ITEMS_KEYS.HELP_CENTER
+    )?.description ?? '';
+
   return (
-    <div className="w-full h-full overflow-hidden">
-      <div className="w-full h-full flex justify-center items-center">
-        <p className="text-center text-lg flex flex-col items-center gap-2 text-raisin-black dark:text-bright-gray">
-          <Help className="w-10 h-10 fill-granite-gray dark:fill-bright-gray" />
-          Please refer to the help center opened in the current browser tab.
-        </p>
-      </div>
-    </div>
+    <ExternalLinkPanel
+      description={description}
+      icon={
+        <Help className="w-10 h-10 fill-granite-gray dark:fill-bright-gray" />
+      }
+    />
```

Comment:
I think this would be more readable

```js
const helpIcon = (
  <Help className="w-10 h-10 fill-granite-gray dark:fill-bright-gray" />
);

<ExternalLinkPanel description={description} icon={helpIcon} />;
```